target/linux/ramips/image/mt7621.mk

```diff
@@ -719,6 +723,25 @@ define Device/lenovo_newifi-d1
 endef
 TARGET_DEVICES += lenovo_newifi-d1
 
+define Device/linksys_e5600
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 26624k
+  DEVICE_VENDOR := Linksys
+  DEVICE_MODEL := E5600
+  DEVICE_PACKAGES := kmod-usb3 kmod-mt7603 kmod-mt7615e \
```

Comment:
If the device has no USB port and no device internally connected by USB , kmod-usb3 package can be dropped and disabled in the dts.